### PR TITLE
Added initial code for Android plugin

### PR DESCRIPTION
**Pull request by dennis-montana**
To test, add our credentials to app.component.html

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,21 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2017 Essent
+nativescript-yourplugin
```
Comment (spike1292) [inline LICENSE]: add plugin & essent name

File: demo/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "tns-core-modules": "2.5.1"
+  },
+  "devDependencies": {
+    "babel-traverse": "6.23.1",
+    "babel-types": "6.23.0",
+    "babylon": "6.15.0",
+    "lazy": "1.0.11",
+    "nativescript-dev-android-snapshot": "^0.*.*",
+    "nativescript-dev-typescript": "~0.3.5",
+    "tns-platform-declarations": "^2.5.0",
```
Comment (spike1292) [inline demo/package.json]: ~2.5.0

File: live-engage.ios.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    public get ios(): UIView {
+        return this._ios;
+    }
+
+    public set ios(value) {
+        this._ios = value;
+    }
+
+    public loadChat(brandId: string, appId: string) {
+        if (!brandId || !appId || !this.ios) {
```
Comment (spike1292) [inline live-engage.ios.ts]: console.error(); which reports whats wrong?

File: LICENSE [hunk 4]
```diff
@@ -1,21 +1,21 @@
-MIT License
+The MIT License (MIT)
```
Comment (code0wl) [inline LICENSE]: Can use the one from Urban airship

File: LICENSE (hunk 1 above)
Comment (dennis-montana) [inline LICENSE]: done

File: LICENSE (hunk 4 above)
Comment (dennis-montana) [inline LICENSE]: done

File: demo/package.json (hunk 2 above)
Comment (dennis-montana) [inline demo/package.json]: done

File: live-engage.ios.ts (hunk 3 above)
Comment (dennis-montana) [inline live-engage.ios.ts]: It isn't really an error if it doesn't continue.

File: platforms/android/include.gradle [hunk 5]
```diff
@@ -0,0 +1,30 @@
+android {
+  productFlavors {
+    "nativescript-live-engage" {
+      dimension "nativescript-live-engage"
+    }
+  }
+}
+
+dependencies {
+
```
Comment (dennis-montana) [inline platforms/android/include.gradle]: done

**Pull request merged**